Docs: Update alerting opt-in page (#35285)
grafana/grafana: docs/sources/alerting/unified-alerting/opt-in.md

Setting the `ngalert` feature toggle enables the new Grafana 8 Alerting system.

> >**Note:** It is recommended to backup Grafana's database before enabling this feature.
> >**Note:** If you use PostgreSQL as a backend database, for using Grafana 8 Alerts, the minimum required version is 9.5.

Reviewer: ```suggestion
>**Note:** It is recommended to backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, the minimum required version is 9.5.
```

Reviewer 2: I suggest also adding a link to instructions for backing up the database.

At startup, when the feature toggle is enabled, Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 Alerting system. You are able to view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your grafana installation.
Also notification channels are migrated to an Alertmanager configuration with the appropriate routes and receivers. Default notification channels are added as contact points to the default route. Notification channels not associated with any dahsboard alert go to the `autogen-unlinked-channel-recv` route.

> `Hipchat` and `Sensu` are discontinued, therefore they  are not migrated to the new alerting. If you have dashboard alerts associated with those type of channels and you want to migrate to the new alerting, make sure you assign a another supported notification channel, so that you continue to receive notification for those alerts.

Reviewer: ```suggestion
Since `Hipchat` and `Sensu` are discontinued, they are not migrated to the new alerting. If you have dashboard alerts associated with those types of channels and you want to migrate to the new alerting, make sure you assign another supported notification channel, so that you continue to receive notification for those alerts.
```
